### DISCUSSION TITLE
ControlNet is image2image ?

### DISCUSSION THREAD

**Discussion by La-fe** [lllyasviel/ControlNet#328] [General]
ControlNet can be abstracted as image to image task, utilizing the generation ability of SD, mapping domain A to domain B through fine-tuning. 

If the meaning of ControlNet can be extended, it can achieve more diverse effects such as deformation and style

**Comment by geroldmeisinger** [comment]
see here [Composer: Creative and Controllable Image Synthesis with Composable Conditions](https://arxiv.org/abs/2302.09778)
